VideoLocker/src/main/java/org/edx/mobile/view/adapters/BaseListAdapter.java

```diff
@@ -125,6 +125,15 @@ public void replace(T item, int position) {
         notifyDataSetChanged();
     }
 
+    public void moveToTop(int itemPosition) {
+        if (itemPosition < objects.size()) {
+            T item = objects.get(itemPosition);
+            objects.remove(itemPosition);
+            objects.add(0, item);
```

Comment:
This can be replaced with:

````java
Collections.swap(objects, 0, itemPosition);
````

And if you're validating the index, then you should validate against the minimum (zero) as well.